std/typecons.d

```diff
@@ -47,11 +47,25 @@ import std.traits;
 debug(Unique) import std.stdio;
 
 /**
-Encapsulates unique ownership of a resource.  Resource of type $(D T) is
-deleted at the end of the scope, unless it is transferred.  The
-transfer can be explicit, by calling $(D release), or implicit, when
-returning Unique from a function. The resource can be a polymorphic
-class object, in which case Unique behaves polymorphically too.
+Encapsulates unique ownership of a resource.
+
+When a $(D Unique!T) goes out of scope it will call $(REF destroy, object)
```

Comment:
Please replace it with $(D object.destroy) if you know or are worried that $(REF destroy, object) isn't going to work.  Thanks!